conf_site/api/views.py

```diff
@@ -1,9 +1,12 @@
 from rest_framework import viewsets
-from rest_framework import response
+from rest_framework import views
+from rest_framework.response import Response
 from rest_framework.permissions import IsAdminUser
+from symposion.conference.models import Conference
 from symposion.speakers.models import Speaker
+from symposion.schedule.models import Presentation
 
-from .serializers import SpeakerSerializer
+from .serializers import *
```

Comment:
Can we avoid using a wildcard import here? It's generally considered a Python antipattern (and is warned against in [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports)).